core/templates/dev/head/css/oppia.css

```diff
@@ -3621,16 +3621,23 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-dashboard-container a,
-.oppia-dashboard-container a:hover,
 .oppia-dashboard-container a:active,
 .oppia-dashboard-container a:visited {
-  color: #009688;
+  color: black;
 }
 
 .oppia-dashboard-container a:hover {
+    color: #009688;
     text-decoration: underline;
 }
 
+.oppia-dashboard-container .oppia-dashboard-feedback,
```

Comment:
Something like .oppia-highlighted-link might be better, given that you're using this class for several different things.